<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>road_inspection_viewer_MainWindow</class>
 <widget class="QMainWindow" name="road_inspection_viewer_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>505</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>505</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">road_inspection_viewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/riv_ico.png</normaloff>../img/riv_ico.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Path_pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">path*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Play_back_pushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Back_pushButton">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>0</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Stop_pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Forward_pushButton">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>0</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Play_forward_pushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delay_pushButton">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>0</y>
      <width>85</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">delay: ??? ms</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save_pushButton">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>0</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Extra_window_pushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>85</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">extra window</string>
    </property>
   </widget>
   <widget class="QPushButton" name="About_pushButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">about</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>25</y>
      <width>640</width>
      <height>478</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Show_pushButton">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>0</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">show</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
